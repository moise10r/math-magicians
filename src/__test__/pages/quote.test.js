import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../components/pages/quote';

it('should match the snapshot of quote', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
