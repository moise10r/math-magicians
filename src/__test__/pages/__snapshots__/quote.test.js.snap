// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot of quote 1`] = `
<section
  className="main-quato-section flex-center"
>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eveniet esse, animi aliquam illo, corrupti atque aliquid neque placeat quisquam tempore qui omnis impedit, laborum hic
     
    <span
      aria-label="img"
      role="img"
    >
      ðŸ¥°ðŸ¥°
    </span>
    perspiciatis vitae temporibus assumenda autem?
    <span
      aria-label="img"
      role="img"
    >
      ðŸ˜ƒðŸ˜ƒ
    </span>
  </p>
</section>
`;
